### PR TITLE
Jagadeesh_Assignment_8_Done

### PR DESCRIPTION
**Pull request by JagadeeshMJun20**
Please Review

### CODE REVIEW

File: src/jagadeeshMJun20/BankingApp.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+	int indCredCount;
+	int indStatementCount;
+		
+		void  debitAmout(){
+		  debitcount++;
+		  indDebCount++; 
+		}
+		void creditAmout(){
+		creditcount++;
+		indCredCount++;
```
Comment (KrishnaKTechnocredits) [inline src/jagadeeshMJun20/BankingApp.java]: You forgot to perform an operation in debitAcc() and credit() METHODS. You should have a balance instance variable. You should write the condition to verify the insufficient balance.

File: src/jagadeeshMJun20/BankingApp.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+		}
+		void printBalance(){
+		statementcount++;
+		indStatementCount++;
```
Comment (KrishnaKTechnocredits) [inline src/jagadeeshMJun20/BankingApp.java]: As method suggest please print data here.

File: src/jagadeeshMJun20/BankingApp.java [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+			System.out.println( userName +" transaction summary : Credit -  " +indCredCount+" times, Debit - "+indDebCount+" times, printBalance - "+indStatementCount+" times");
+		}	
+		void allTransactionSummary() {
+			System.out.println("All transaction summary   : Credit - " +creditcount+" times, Debit - "+debitcount+" times, printBalance - "+statementcount+" times" );
+		}
+		public static void main(String[] agrs) {
+			BankingApp bankingApp1 = new BankingApp();
+			BankingApp bankingApp2 = new BankingApp();
+			bankingApp1.debitAmout();
+			bankingApp1.creditAmout();
```
Comment (KrishnaKTechnocredits) [inline src/jagadeeshMJun20/BankingApp.java]: Where is the amount.

**Review by JagadeeshMJun20 — COMMENTED** [review]
Please review the code changes.

File: src/jagadeeshMJun20/BankingApp.java (hunk 1 above)
Comment (JagadeeshMJun20) [reply src/jagadeeshMJun20/BankingApp.java]: Updated the code as per the comments.

**Review by KrishnaKTechnocredits — COMMENTED** [review]
Code looks good to me.

**Pull request merged**